.github/workflows/ci.yml

```diff
@@ -44,6 +44,10 @@ jobs:
           rustup default ${{ matrix.rust }}
       - name: Build
         run: make program CARGOFLAGS="--verbose"
+      - name: Insert zstd module
```

Comment:
Can you move it up right after "Insert zram module"? I think it's nicer to keep those two together.